notes-apps-ogechukwu-okoli.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+class Note:#base class
+    '''
+    It initializes new notes
+    '''
+    def __init__(self, note_id, content, created_at):
+        self.note_id = note_id # Notes_id
+        self.content = content # it contains the contents in the notes
+        self.created_at = created_at #store the created time stamp
+class TextNote(Note): #Child class
```

Comment:
Every class and every function should have a docstring that describes what that function is doing